test/functional/data.spec.js

```diff
@@ -0,0 +1,52 @@
+const got = require('got');
+const {expect} = require('chai');
+
+let data;
+
+function checkURL(url, head = false) {
+	const method = head ? 'head' : 'get';
+	return got[method](url, {throwHttpErrors: false}).then(({statusCode}) => {
+		if (statusCode === 403) {
```

Comment:
```suggestion
		if (!head && statusCode === 403) {
```

I think we should add a guard, otherwise it could infinite loop and crash